db/bigtable_eth1.go

```diff
@@ -3291,9 +3300,11 @@ func (bigtable *Bigtable) GetTokenTransactionsTableData(token []byte, address []
 	names := make(map[string]string)
 	tokens := make(map[string]*types.ERC20Metadata)
 	for _, t := range transactions {
-		names[string(t.From)] = ""
-		names[string(t.To)] = ""
-		tokens[string(t.TokenAddress)] = nil
+		if t != nil {
```

Comment:
This check does not suffice, `t` is accessed a few times below without protection, see here:
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/2f7566c921508e73a435184ee5d86c78258e95f3/db/bigtable_eth1.go#L3318